plugins/pitchfork.plug

```diff
@@ -1,6 +1,7 @@
 [Core]
-name = pitchfork
+name = Pitchfork
```

Comment:
why change name?

Reply:
To make it workable with CoroboTestCase. According to errbot documentation plugin name and the class name should be same.